docs/ambiorix/07-response.md

```diff
@@ -74,27 +81,17 @@ app$get("/:book", \(req, res){
 
 ## Status
 
-The status of the response can be specified in the response method (e.g.: `render('home', status = 200L)`), or with the `status` method.
+The HTTP status of the response can be specified using the `set_status()` method.
 
 ```r
 app$get("/error", \(req, res){
-  res$status(500)
+  res$set_status(500L)
```

Comment:
we could change that to the setter?

https://github.com/ambiorix-web/ambiorix/blob/b23dc4e6a549cbc543ca9938b09d56a11be7e0ae/R/response.R#L578

```r
res$status <- 500
```

I find it more readable

Reply:
ah yes, agreed.
i'm actually discovering that now :/